apps/s_server.c

```diff
@@ -958,6 +959,7 @@ const OPTIONS s_server_options[] = {
     {"early_data", OPT_EARLY_DATA, '-', "Attempt to read early data"},
     {"num_tickets", OPT_S_NUM_TICKETS, 'n',
      "The number of TLSv1.3 session tickets that a server will automatically  issue" },
+    {"no_anti_replay", OPT_NO_ANTI_REPLAY, '-', "Switch of anti-replay protection"},
```

Comment:
Switch-off

Reply:
No hyphen, though, since switch is the verb: "Switch off anti-replay protection".

Reply:
Fixed.